firefox_media_tests/playback/netflix/test_basic_playback.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from media_test_harness.testcase import MediaTestCase
+from firefox_media_tests.playback.video_playback_tests_mixin import (
+    VideoPlaybackTestsMixin
+)
+
+
+class TestBasicNetflixPlayback(MediaTestCase,

```

Comment:
As discussed, since only the test-urls are specific to Netflix, the file/class/directory names should not refer to Netflix. I'd put this in `playback/test_eme_playback.py` and call the class `TestEMEPlayback` or something along those lines.